README.md

```diff
@@ -83,15 +83,17 @@ fail due to code changes. You can just run:
  
     ./gradlew processMessages processTestMessages
 
+### Running a Kafka broker in KRaft mode
+
+    KAFKA_CLUSTER_ID="$(./bin/kafka-storage.sh random-uuid)"
+    ./bin/kafka-storage.sh format -t $KAFKA_CLUSTER_ID -c config/kraft/server.properties
+    ./bin/kafka-server-start.sh config/kraft/server.properties
+
```

Comment:
Can we have a `###` heading with something like "Running a Kafka Broker" with 2 `####` heading, one for KRaft-mode and one for ZK-mode?

The `### Running a Kafka Broker` should mention that only one subsection should be followed. In other words the Kafka cluster should have one KRaft-mode server or ZK-mode server but not both.

Reply:
For better or worse, the README is not structured as a quick start. You are not supposed to follow every step in it. Instead, it is meant as a useful set of things one may want to do. The thing you suggested doesn't fit with everything else we have in my opinion. What do you think?